rotkehlchen/data/importer.py

```diff
@@ -934,3 +934,67 @@ def import_nexo_csv(self, filepath: Path) -> Tuple[bool, str]:
                 except KeyError as e:
                     return False, str(e)
         return True, ''
+
+    def _consume_shapeshift_trade(self, csv_row: Dict[str, Any]) -> None:
+        """
+        Consume the file containing only trades from ShapeShift.
+        """
+        timestamp = deserialize_timestamp_from_date(
+            date=csv_row['timestamp'],
+            formatstr='iso8601',
+            location='ShapeShift',
+        )
+        buy_asset = symbol_to_asset_or_token(csv_row['outputCurrency'])
+        buy_amount = deserialize_asset_amount(csv_row['outputAmount'])
+        sold_asset = symbol_to_asset_or_token(csv_row['inputCurrency'])
+        sold_amount = deserialize_asset_amount(csv_row['inputAmount'])
+        rate = deserialize_asset_amount(csv_row['rate'])
+        fee = deserialize_fee(csv_row['minerFee'])
+        in_addr = csv_row['inputAddress']
+        out_addr = csv_row['outputAddress']
+
+        if sold_amount == ZERO:
+            log.debug(f'Ignoring ShapeShift trade with sold_amount equal to zero. {csv_row}')
+            return
+        if in_addr == '' or out_addr == '':
+            log.debug(f'Ignoring ShapeShift trade which was performed on DEX. {csv_row}')
+            return
+        trade = Trade(
+            timestamp=timestamp,
+            location=Location.SHAPESHIFT,
+            base_asset=buy_asset,
+            quote_asset=sold_asset,
+            trade_type=TradeType.BUY,
+            amount=buy_amount,
+            rate=Price(rate),
+            fee=Fee(fee),
+            fee_currency=buy_asset,  # Assumption that mineFee is denominated in outputCurrency
```

Comment:
I assume typo

```suggestion
            fee_currency=buy_asset,  # Assumption that minFee is denominated in outputCurrency
```

Reply:
it is actually `minerFee`.  Wanted to discuss this.  It is not technically an exchange fee, and as such, does it belong reported with the trade as a Fee?  `outputAmount = inputAmount * rate - minerFee`